openpype/pipeline/delivery.py

```diff
@@ -299,10 +336,28 @@ def hash_path_exist(myPath):
             os.path.join(dir_path, src_file_name)
         )
 
-        dst_padding = dst_collection.format("{padding}") % index
-        dst = "{}{}{}".format(dst_head, dst_padding, dst_tail)
-        log.debug("Copying single: {} -> {}".format(src, dst))
-        _copy_file(src, dst)
+        if renumber_frame:
+            dsp_index = (re_frame_value - int(index)) + 1
```

Comment:
Typo in variable `dsp_index` -> `dst_index`.

And I'm not sure I understand the logic? If `re_frame_value` is `1001` and source frame is `1001` the output frame would be `1`.